MIT License

Copyright (c) 2017 Stanford Vision and Learning Group

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.

Date: Feb 06 21:53:34 2019

From: 8faf9c60a22b60d511041f338c9ff64fcf137e57 Feb 20 16:34:11 2019

Subject: [PATCH] XONE-19941 : ntpd crash on pthread warmup call

	Disable pthread warmup api as it is causing thread unwinding on pthread_cancel

Source: COMCAST

Upstream-Status: Pending

Signed-off-by: mselva006c <mani_selvaraj@comcast.com>
---

Index: ntp-4.2.8p11/ntpd/ntpd.c
===================================================================
--- ntp-4.2.8p11.orig/ntpd/ntpd.c
+++ ntp-4.2.8p11/ntpd/ntpd.c
@@ -34,9 +34,6 @@
 # ifdef HAVE_PTHREAD_H
 #  include <pthread.h>
 # endif
-# if defined(linux)
-#  define NEED_PTHREAD_WARMUP
-# endif
 #endif
 
 #ifdef HAVE_UNISTD_H
